src/main/java/org/broadinstitute/consent/http/models/PaginationToken.java

```diff
@@ -131,20 +130,30 @@ public String toBase64() {
     return Base64.getEncoder().encodeToString(new Gson().toJson(this).getBytes(UTF_8));
   }
 
-  private void checkSortField(String sortField) {
-    if (Objects.nonNull(sortField)) {
-      if (!acceptableSortFields.contains(sortField)) {
-        throw new BadRequestException("Cannot sort on given field: " + sortField);
-      }
+  public void setAcceptableSortField(Map<String, String> acceptableSortFields) {
+    this.acceptableSortFields = acceptableSortFields;
+  }
+
+  public Map<String, String> getAcceptableSortFields() {
+    return acceptableSortFields;
+  }
+
+  private String validateSortField(String sortField) {
+    if(Objects.isNull(sortField) || Objects.isNull(acceptableSortFields.get(sortField))) {
+      return "dar_code";
     }
```

Comment:
I think I missed this on the first pass. Having a default of `dar_code` ties this class directly to `DarCollection`. I think it might be better to throw an exception here if we really can't validate it. Another thought is to add a default field as a second argument, but that seems a little convoluted since this is only used in the constructor.